scripts/hypothesistooling.py

```diff
@@ -82,6 +82,12 @@ def hash_for_name(name):
     ]).decode('ascii').strip()
 
 
+def is_ancestor(a, b):
+    return subprocess.call([
+        'git', '--is-ancestor', a, b
```

Comment:
I should have been more explicit about the problem – the command you’ve implemented doesn’t include the `merge-base` verb.